tests/contrib/test_kubernetes/test_base.py

```diff
@@ -0,0 +1,124 @@
+import os
+import re
+import subprocess
+import tempfile
+import time
+
+import numpy as np
+import pytest
+from kubernetes import config
+from sklearn.datasets import load_iris
+
+from mlem.config import project_config
+from mlem.contrib.docker.base import DockerDaemon, DockerEnv
+from mlem.contrib.docker.helpers import build_model_image
+from mlem.contrib.kubernetes.base import (
+    K8sDeployment,
+    K8sDeploymentState,
+    K8sEnv,
+)
+from mlem.core.objects import DeployStatus
+from tests.contrib.test_kubernetes.conftest import k8s_test
+from tests.contrib.test_kubernetes.utils import Command
+
+
+@pytest.fixture
+def minikube_env_variables():
+    output = subprocess.check_output(
+        ["minikube", "-p", "minikube", "docker-env"]
+    )
+    export_re = re.compile('export ([A-Z_]+)="(.*)"\\n')
+    export_pairs = export_re.findall(output.decode("UTF-8"))
+    for k, v in export_pairs:
+        os.environ[k] = v
+
+
+@pytest.fixture
+def load_kube_config():
+    config.load_kube_config(os.getenv("KUBECONFIG", default="~/.kube/config"))
+
+
+@pytest.fixture
+def k8s_deployment(minikube_env_variables, model_meta_saved_single):
+    return K8sDeployment(
+        name="ml",
+        model=model_meta_saved_single.make_link(),
+        image_pull_policy="Never",
+        service_type="LoadBalancer",
+        daemon=DockerDaemon(host=os.getenv("DOCKER_HOST", default="")),
+    )
+
+
+@pytest.fixture
+def docker_image(k8s_deployment):
+    tmpdir = tempfile.mkdtemp()
+    k8s_deployment.dump(os.path.join(tmpdir, "deploy"))
+    return build_model_image(
+        k8s_deployment.get_model(),
+        k8s_deployment.image_name,
+        k8s_deployment.server
+        or project_config(
+            k8s_deployment.loc.project if k8s_deployment.is_saved else None
```

Comment:
it cant be saved since you just created it

Reply:
removed, passing only `None` now